vector/src/main/scala/geotrellis/vector/Point.scala

```diff
@@ -31,6 +31,9 @@ object Point {
 
   implicit def jtsCoord2Point(coord: jts.Coordinate): Point =
     Point(factory.createPoint(coord))
+
+  implicit def pointToTuple2(point: Point): (Double, Double) =
```

Comment:
While this is a nice convenience for people familiar with the particular code that translates between `Point` and `Tuple2`, I've been burned enough by unexpected or unintended conversions in the past to avoid this sort of thing personally, especially when the conversion is to a more general type. If you were converting between `Point` and something like `Position(lat, lng)` I'd be a bit less concerned because the translation semantics of `x` -> `lat` and `y` -> `lng` are clear, but `_1` and `_2` have no inherent semantics and therefore information is lost in the implicit conversion.

Reply:
Yes, i decided to remove this implicit at all as it is indeed ambiguous.